code/modules/organs/subtypes/diona.dm

```diff
@@ -178,8 +178,22 @@
 	dislocated = -1
 	joint = "structural ligament"
 	amputation_point = "branch"
+	vital = FALSE // Lore team requested this, not vital organ. We can still live without it.
 
+/obj/item/organ/external/head/diona/New(loc, ...)
```

Comment:
what?
Why are you:
- Using New instead of Initialize()
- Not using whatever organ-attach proc surely exists